extension/src/standalone_pages/InstructorDashboardRenderVideoInfo.js

```diff
@@ -39,4 +75,146 @@ function renderVideoAccordian(document, videoObjects) {
     document.getElementById("nav-videos").appendChild(accordianDiv);
 }
 
-export { renderVideoAccordian };
+function renderActiveFeedback(video, response) {
+    let color= "#5959e6";
+    let activeChart = document.getElementById(`activeFeedback_${video.videoID}`).getContext("2d");
+    response.active_questions.sort((a, b) => {
+        return a.timestamp - b.timestamp;
+    });
+    let labels = response.active_questions.flatMap((question) => {
```

Comment:
let's try a different way with `filter` instead of double `flatMap`
```
response.active_question.flat((e) => {
    if (question.timestamp && question.count) {
        return true;
    }
}
```

Any particular reason with filtering based on the video duration? If your concern is on the integrity of the test data, please see my comment in the test google classroom about deprecation and migration.

Reply:
We need flatmap to reformat the data.
Chart.js requires the data to be in an array [1,2,3,4].
Filter can only eliminate the data and cannot reformat the data from [{timestamp: 1, count: 1}] -> [1]

Reply:
We need flatmap to reformat the data.
Chart.js requires the data to be in an array [1,2,3,4].
Filter can only eliminate the data and cannot reformat the data from [{timestamp: 1, count: 1}] -> [1]

Also, I'm not sure what you're referring to about "test  google classroom"

Reply:
I meant to say `filter` and then `map`. So with `filter`, we can get rid of ones that are invalid. Then, we can use `map` in a more readable way to extract out the datapoints (or transform the datapoints like you mentioned)

Reply:
This is [test google classroom](https://classroom.google.com/u/0/c/MTYwMDYwNzIzNTU2). The name is Testing 123

Reply:
So we actually need to split the data as well.
{timestamp: 1, count: 1}.
 I don't believe we can do this with filter and map.
I can try something with lodash.partition but at this point I don't think it's worth it.

I would prefer to re-visit this after the semester is over when I have more time.